rules/prefer-modern-dom-apis.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+const getDocsUrl = require('./utils/get-docs-url');
+
+const checkForReplaceChildOrInsertBefore = (context, node) => {
+	const getArgumentName = args => {
+		if (args.type === 'Identifier') {
+			return args.name;
+		}
+
+		return null;
+	};
+
+	const forbiddenIdentifierNames = new Map([
+		['replaceChild', 'replaceWith'],
+		['insertBefore', 'before']
+	]);
+
+	const identifierName = node.callee.property.name;
+
+	// Only handle methods that exist in forbiddenIdentifierNames
+	if (forbiddenIdentifierNames.has(identifierName)) {
```

Comment:
I prefer the return-early when the condition is being used as a guard.

```
	if (!forbiddenIdentifierNames.has(identifierName)) {
		return;
	}
```

Reply:
Good point, changing it in the upcoming commit.